nodejs/clients.js

```diff
@@ -1,22 +1,15 @@
-const { readMasterKey, CsfleHelper } = require("./helpers")
+const kms = require("./kms");
+require("dotenv").config();
 
-const localMasterKey = readMasterKey()
-
-const csfleHelper = new CsfleHelper({
-  kmsProviders: {
-    local: {
-      key: localMasterKey
-    }
-  }
-})
+const kmsClient = kms.localCsfleHelper();
 
 async function main() {
   // change this to the base64 encoded data key generated from make-data-key.js
-  let dataKey = null // change this!
+  let dataKey = null; // change this!
 
-  let regularClient = await csfleHelper.getRegularClient()
-  let schemeMap = csfleHelper.createJsonSchemaMap(dataKey)
-  let csfleClient = await csfleHelper.getCsfleEnabledClient(schemeMap)
+  let regularClient = await kmsClient.getRegularClient();
+  let schemeMap = kmsClient.createJsonSchemaMap(dataKey);
+  let csfleClient = await kmsClient.getCsfleEnabledClient(schemeMap);
```

Comment:
Did you mean `schemaMap` rather than `schemeMap`?
```suggestion
  let schemaMap = kmsClient.createJsonSchemaMap(dataKey);
  let csfleClient = await kmsClient.getCsfleEnabledClient(schemaMap);
```